src/commands/init.js

```diff
@@ -35,16 +35,46 @@ export async function init({ flags }) {
 		process.exit(1);
 	}
 
-	const data = await prompt([name, lang, main_dir, cmds_dir, default_prefix]);
+	initProject(isDefault);
+}
+
+/**
+ * Initialize a new project, either with prompts or with default values.
+ * @param {boolean} isDefault - Whether or not to use default values.
+ */
+const initProject = async (isDefault) => {
+	let data;
+
+	if (!isDefault) {
+		data = await prompt([name, lang, main_dir, cmds_dir, default_prefix]);
+	} else {
+		const projectName = await prompt([name]);
+
+		data = {
+			name: projectName.name,
+			lang: 'typescript',
+			main_dir: 'src',
+			cmds_dir: 'commands',
+			default_prefix: '!',
+		};
+	}
 
-	if (Object.keys(data).length < 5) process.exit(1);
+	if (!isDefault && Object.keys(data).length < 5) process.exit(1);
```

Comment:
why `!isDefault` is needed?
even when the defaults are used, data object is getting all 5 properties needed